### PR TITLE
fix(): routing for custom domains + nested routes

### PR DESCRIPTION
**Pull request by harshmaur**
Changes proposed in this pull request:
- create multiple routes for home and space routes depending of whether its a custom domain or not. 
- Let the space view handle the aliases instead of implementing beforeEnter (moreover beforeEnter does not work well for children routes)
- changes named home routes to / path routes (I do not know why the named routes didn't work as expected)

### CODE REVIEW

**Comment by bonustrack** [comment]
@harshmaur The urls with alias don't work for me
http://localhost:3000/#/balancer
http://localhost:3000/#/balancer/proposal/QmfQL7iBcQ5yFVsADdHij5TgYheFkqiJGFQk2xBkkZiJS9
Also these route should be redirected to the correct space id, the name "alias" is a bit confusing but this define the old => new space id, the old space id should not be visible on the url. If it's on a custom domain you can redirect to the proposal url without space id direct like 
http://localhost:3000/#/proposal/QmfQL7iBcQ5yFVsADdHij5TgYheFkqiJGFQk2xBkkZiJS9

**Comment by harshmaur** [comment]
@bonustrack thanks for the examples. I'll check.

**Comment by harshmaur** [comment]
@samuveth I need your help merging this because there was a major refactor and unreadIndicator work is also included now in develop

**Comment by samuveth** [comment]
> @samuveth I need your help merging this because there was a major refactor and unreadIndicator work is also included now in develop

Ok will do that now.

File: src/views/SpaceCreate.vue [hunk 1]
```diff
@@ -191,7 +190,7 @@ onMounted(async () => {
     <template #content-left>
       <div class="px-4 md:px-0 mb-3">
         <router-link
-          :to="{ name: domain ? 'home' : 'proposals' }"
```
Comment (samuveth) [inline src/views/SpaceCreate.vue:194]: Somehow on going back from `/create` on custom domain doesn't work

Comment (samuveth) [reply src/views/SpaceCreate.vue:194]: Otherwise works great!

**Comment by harshmaur** [comment]
@samuveth problem resolved.

**Review by samuveth — APPROVED** [review]
Works very well for me!

**Comment by harshmaur** [comment]
@bonustrack good to merge?

**Review by bonustrack — APPROVED** [review]

**Pull request merged**